js/controller.js

```diff
@@ -642,6 +642,8 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.state.upNextInfo.delayedSetEmbedCodeEvent = true;
       }
       else {
+        this.state.screenToShow = CONSTANTS.SCREEN.LOADING_SCREEN;
+        this.renderSkin();
```

Comment:
What would happen if we move renderSkin() after the 2 events publish below? I.e does renderSkin() need to be called here?